string item_colour
uint8 zone
int16 zone_x
int16 zone_y